mlperf_logging/package_checker/package_checker.py

```diff
@@ -161,11 +179,12 @@ def check_training_result_files(folder, usage, ruleset, quiet, werror,
             'Found too many errors in logging, see log above for details.')
 
 
-def check_systems(folder, ruleset):
+def check_systems(folder, usage, ruleset):
     """Checks the system decription files
 
     Args:
         folder: The folder for a submission package.
+        usage: The usage such as training, inference_edge, inference_server, hpc.
```

Comment:
Do we use this for inference too?

Reply:
I don't think so, but `inference_edge, inference_server,` were there before actually; I didn't add those.

I can remove them if indeed inference is not in the scope of mlcommons/logging.